packages/block-library/src/button/edit.js

```diff
@@ -72,8 +101,20 @@ class ButtonEdit extends Component {
 			text,
 			url,
 			title,
+			linkTarget,
+			rel,
+			align,
 		} = attributes;
 
+		let PopoverPosition;
```

Comment:
Just a capitalisation issue here, it should be `popoverPosition` according to the coding standards.

Reply:
Whoops, missed it! Thank you for noticing, fixed in [this commit](https://github.com/WordPress/gutenberg/pull/10128/commits/d76301719a97c11a86d513e2ce49f9406c785a6e) ❤️